io.github.monsterbirdlimited.soscap.yaml

```diff
@@ -0,0 +1,81 @@
+---
+app-id: io.github.monsterbirdlimited.soscap
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+command: soscap
+finish-args:
+  - --filesystem=home
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --device=dri
+  - --talk-name=org.freedesktop.FileManager1
+  - --filesystem=xdg-videos
+  - --env=G_MESSAGES_DEBUG=none
+  - --env=GST_DEBUG=3
```

Comment:
why do you need debug ?

Reply:
I do not anymore and will remove. I was never able to get it working properly in testing.